internal/util.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "strings"
+
+func IsOneOf(list []string, s string, caseSensitive bool) bool {
```

Comment:
How about renaming this function to `ContainsString` and removing the `caseSensitive` parameter?

We can introduce a new `ContainsStringCaseInsensitive` function which checks compares with case sensitivity.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/41/commits/0bdfd09792e7af0022e3b8ab244bec7b2813bd57

Will introduce case insensitive version when it is needed 👍